chipy_org/apps/profiles/migrations/0004_rsvp_migration.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+import probablepeople
+
```

Comment:
How will this work when I kill the probablepeople? Does it need to be wrapped in try, except?